tests/foreman/cli/test_import.py

```diff
@@ -315,14 +341,56 @@ def test_import_users_default(self, data):
         })
         os.remove(new_dataset)
         ssh.command(
-            'rm -rf ${{HOME}}/.transition_data {0} {1}'
-            .format(pwdfile, tmp_dir)
+            'rm -rf ${{HOME}}/.transition_data {}'
+            .format(pwdfile)
         )
         self.assertEqual(
             ssh_import.stdout,
             [u'Summary', u'  Created 3 users.', u'']
         )
 
+    @data(*positive_import_user_data())
+    def test_reimport_users_default(self, data):
```

Comment:
I ran pylint over this file, and it's giving me warnings like this for several test methods:

    W:353,42: Redefining name 'data' from outer scope (line 10) (redefined-outer-name)
    W:353,42: Unused argument 'data' (unused-argument)

and it seems to be correct. I don't see `data` being used anywhere here. What's the deal? Is the `@data` decorator even necessary? And assuming it is, can we rename the method argument to something that doesn't conflict?